app/src/main/java/com/example/grosharies/ui/navigation/Screen.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.grosharies.ui.navigation
+
+sealed class Screen(val route: String) {
+    object Home : Screen("home/")
```

Comment:
How'd you get to this approach? Providing a full route seems more like a compose web thing or support for deeplinking though. Don't think this is necessary. 

Only thing is, you don't really seem to need a sealed class for this. Might as well been an object/ sealed interface. Since you don't do when checks on it, but just access it directly.